### DISCUSSION TITLE
Is it possible to correspond the electrode stimulation target to the brain atlas mapping brain region?

### DISCUSSION THREAD

**Discussion by xiongyy99255** [simnibs/simnibs#259] [9. Results and analysis]
Dear SimNIBS team,
      I have a little problem with the analysis using simNIBS, how do I know, based on the electrode stimulation target (e.g. F6), to which brain region of the HCP atlas my stimulation target should correspond to after interpolating to the FsAverage space (e.g., use the HCP-MMP1)? Or how should I look for the mapping relationship among them?
Thanks for your consideration and for the continuous effort with SimNIBS,

![1702209742922](https://github.com/simnibs/simnibs/assets/129670513/eff30e42-bb29-40e2-aae1-ef87a7dfbce2)

**Comment by Kstoffer** [comment]
The electrode coordinates in the simNIBS folder (resources/ElectrodeCaps_MNI/EEG10-10_UI_Jurak_2007.csv) are in MNI152 space, which is fairly close to the fsaverage space, so you could just check that coordinate.
You can also convert this coordinate to fsaverage space (see freesurfer tutorials) but the change will be fairly small.
If you want to know the main region stimulated in fsaverage space you can convert coordinates via subject2mni: https://simnibs.github.io/simnibs/build/html/documentation/command_line/subject2mni_coords.html
Is this helping?
